mris_congeal/mris_congeal.cpp

```diff
@@ -486,7 +486,7 @@ get_option(int argc, char *argv[])
            "and using it for initial alignment\n", inflated_name) ;
     sprintf(fname, "%s.H", argv[2]) ;
     curvature_names[0] = (char *)calloc(strlen(fname)+1, sizeof(char)) ;
-    strcpy(curvature_names[0], fname) ;
+    strcpy(const_cast<char*>(curvature_names[0]), fname) ; // const_cast AND strcpy
```

Comment:
Note that `curvature_names` is an array of character arrays declared and initialized above. If `fname` is ever longer than the first entry in this array, I believe the `strcpy()` is going to clobber something random. In any case, I believe that it's going to clobber the original constant initialisation.